{"uid":"7dbfeca1528ebbb8","name":"Создание простой модели, проверка и удаление","fullName":"Test_Cases.test_model.TestModel#test_model","historyId":"86380ec227ad971d2d967b7352fed79e","time":{"start":1691060030479,"stop":1691060030479,"duration":0},"description":"Проверка работоспособности раздела 'Модели'","descriptionHtml":"<p>Проверка работоспособности раздела 'Модели'</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x562f731f1783 <unknown>\n#1 0x562f72f1d917 <unknown>\n#2 0x562f72f48978 <unknown>\n#3 0x562f72f459c9 <unknown>\n#4 0x562f72f85a13 <unknown>\n#5 0x562f72f8515f <unknown>\n#6 0x562f72f7cbd3 <unknown>\n#7 0x562f72f50a7b <unknown>\n#8 0x562f72f5181e <unknown>\n#9 0x562f731b2e98 <unknown>\n#10 0x562f731b6d67 <unknown>\n#11 0x562f731c132c <unknown>\n#12 0x562f731b7993 <unknown>\n#13 0x562f73186827 <unknown>\n#14 0x562f731db8a8 <unknown>\n#15 0x562f731dba36 <unknown>\n#16 0x562f731ea5f3 <unknown>\n#17 0x7fb7173f7b43 <unknown>","statusTrace":"request = <SubRequest 'get_webdriver' for <Function test_homepage>>\nget_chrome_options = <selenium.webdriver.chrome.options.Options object at 0x7f5325e08100>\n\n    @pytest.fixture(scope='session')\n    def get_webdriver(request, get_chrome_options):\n        options = get_chrome_options\n        #driver = webdriver.Chrome(ChromeDriverManager().install(), options=options)\n>       driver = webdriver.Chrome(options=options)\n\nTest_Cases/conftest.py:35: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py:49: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py:54: in __init__\n    super().__init__(\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:206: in __init__\n    self.start_session(capabilities)\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:291: in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:346: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f5325e08940>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: Chrome failed to start: exited a...\\\\n#15 0x562f731dba36 \\\\u003Cunknown>\\\\n#16 0x562f731ea5f3 \\\\u003Cunknown>\\\\n#17 0x7fb7173f7b43 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        if status in ErrorCode.NO_SUCH_ELEMENT:\n            exception_class = NoSuchElementException\n        elif status in ErrorCode.NO_SUCH_FRAME:\n            exception_class = NoSuchFrameException\n        elif status in ErrorCode.NO_SUCH_SHADOW_ROOT:\n            exception_class = NoSuchShadowRootException\n        elif status in ErrorCode.NO_SUCH_WINDOW:\n            exception_class = NoSuchWindowException\n        elif status in ErrorCode.STALE_ELEMENT_REFERENCE:\n            exception_class = StaleElementReferenceException\n        elif status in ErrorCode.ELEMENT_NOT_VISIBLE:\n            exception_class = ElementNotVisibleException\n        elif status in ErrorCode.INVALID_ELEMENT_STATE:\n            exception_class = InvalidElementStateException\n        elif (\n            status in ErrorCode.INVALID_SELECTOR\n            or status in ErrorCode.INVALID_XPATH_SELECTOR\n            or status in ErrorCode.INVALID_XPATH_SELECTOR_RETURN_TYPER\n        ):\n            exception_class = InvalidSelectorException\n        elif status in ErrorCode.ELEMENT_IS_NOT_SELECTABLE:\n            exception_class = ElementNotSelectableException\n        elif status in ErrorCode.ELEMENT_NOT_INTERACTABLE:\n            exception_class = ElementNotInteractableException\n        elif status in ErrorCode.INVALID_COOKIE_DOMAIN:\n            exception_class = InvalidCookieDomainException\n        elif status in ErrorCode.UNABLE_TO_SET_COOKIE:\n            exception_class = UnableToSetCookieException\n        elif status in ErrorCode.TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.SCRIPT_TIMEOUT:\n            exception_class = TimeoutException\n        elif status in ErrorCode.UNKNOWN_ERROR:\n            exception_class = WebDriverException\n        elif status in ErrorCode.UNEXPECTED_ALERT_OPEN:\n            exception_class = UnexpectedAlertPresentException\n        elif status in ErrorCode.NO_ALERT_OPEN:\n            exception_class = NoAlertPresentException\n        elif status in ErrorCode.IME_NOT_AVAILABLE:\n            exception_class = ImeNotAvailableException\n        elif status in ErrorCode.IME_ENGINE_ACTIVATION_FAILED:\n            exception_class = ImeActivationFailedException\n        elif status in ErrorCode.MOVE_TARGET_OUT_OF_BOUNDS:\n            exception_class = MoveTargetOutOfBoundsException\n        elif status in ErrorCode.JAVASCRIPT_ERROR:\n            exception_class = JavascriptException\n        elif status in ErrorCode.SESSION_NOT_CREATED:\n            exception_class = SessionNotCreatedException\n        elif status in ErrorCode.INVALID_ARGUMENT:\n            exception_class = InvalidArgumentException\n        elif status in ErrorCode.NO_SUCH_COOKIE:\n            exception_class = NoSuchCookieException\n        elif status in ErrorCode.UNABLE_TO_CAPTURE_SCREEN:\n            exception_class = ScreenshotException\n        elif status in ErrorCode.ELEMENT_CLICK_INTERCEPTED:\n            exception_class = ElementClickInterceptedException\n        elif status in ErrorCode.INSECURE_CERTIFICATE:\n            exception_class = InsecureCertificateException\n        elif status in ErrorCode.INVALID_COORDINATES:\n            exception_class = InvalidCoordinatesException\n        elif status in ErrorCode.INVALID_SESSION_ID:\n            exception_class = InvalidSessionIdException\n        elif status in ErrorCode.UNKNOWN_METHOD:\n            exception_class = UnknownMethodException\n        else:\n            exception_class = WebDriverException\n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\nE         (unknown error: DevToolsActivePort file doesn't exist)\nE         (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nE       Stacktrace:\nE       #0 0x562f731f1783 <unknown>\nE       #1 0x562f72f1d917 <unknown>\nE       #2 0x562f72f48978 <unknown>\nE       #3 0x562f72f459c9 <unknown>\nE       #4 0x562f72f85a13 <unknown>\nE       #5 0x562f72f8515f <unknown>\nE       #6 0x562f72f7cbd3 <unknown>\nE       #7 0x562f72f50a7b <unknown>\nE       #8 0x562f72f5181e <unknown>\nE       #9 0x562f731b2e98 <unknown>\nE       #10 0x562f731b6d67 <unknown>\nE       #11 0x562f731c132c <unknown>\nE       #12 0x562f731b7993 <unknown>\nE       #13 0x562f73186827 <unknown>\nE       #14 0x562f731db8a8 <unknown>\nE       #15 0x562f731dba36 <unknown>\nE       #16 0x562f731ea5f3 <unknown>\nE       #17 0x7fb7173f7b43 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py:245: WebDriverException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_webdriver","time":{"start":1691059996247,"stop":1691060030326,"duration":34079},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x562f731f1783 <unknown>\n#1 0x562f72f1d917 <unknown>\n#2 0x562f72f48978 <unknown>\n#3 0x562f72f459c9 <unknown>\n#4 0x562f72f85a13 <unknown>\n#5 0x562f72f8515f <unknown>\n#6 0x562f72f7cbd3 <unknown>\n#7 0x562f72f50a7b <unknown>\n#8 0x562f72f5181e <unknown>\n#9 0x562f731b2e98 <unknown>\n#10 0x562f731b6d67 <unknown>\n#11 0x562f731c132c <unknown>\n#12 0x562f731b7993 <unknown>\n#13 0x562f73186827 <unknown>\n#14 0x562f731db8a8 <unknown>\n#15 0x562f731dba36 <unknown>\n#16 0x562f731ea5f3 <unknown>\n#17 0x7fb7173f7b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/runner/work/LTA_TESTS/LTA_TESTS/Test_Cases/conftest.py\", line 35, in get_webdriver\n    driver = webdriver.Chrome(options=options)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.10/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true,"stepsCount":0},{"name":"get_chrome_options","time":{"start":1691059996247,"stop":1691059996247,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false,"stepsCount":0}],"afterStages":[],"labels":[{"name":"parentSuite","value":"Test_Cases"},{"name":"suite","value":"test_model"},{"name":"subSuite","value":"TestModel"},{"name":"host","value":"fv-az453-86"},{"name":"thread","value":"2234-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"Test_Cases.test_model"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":11,"skipped":0,"passed":6,"unknown":0,"total":17},"items":[{"uid":"3ce23c253d7f8e16","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/43//#testresult/3ce23c253d7f8e16","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=45861): Max retries exceeded with url: /session/e5a689c724648f0d5308caa4a8bd27b1/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f87d31b2d10>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1691045250612,"stop":1691045250612,"duration":0}},{"uid":"ee8ff1c2869ce82b","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/43//#testresult/ee8ff1c2869ce82b","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=59593): Max retries exceeded with url: /session/9568bcb038578a21277e87dd4550c15b/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ff60fd6ec50>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1690989527174,"stop":1690989527174,"duration":0}},{"uid":"716a075ef457aff0","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/42//#testresult/716a075ef457aff0","status":"broken","statusDetails":"urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=45237): Max retries exceeded with url: /session/ac81092ea0663ab08d44674463c37387/element (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdf2173f7f0>: Failed to establish a new connection: [Errno 111] Connection refused'))","time":{"start":1690988222521,"stop":1690988222521,"duration":0}},{"uid":"a694b68929686828","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/41//#testresult/a694b68929686828","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/app-root/app-auth-screen/app-login-screen/form/div/lta-input[1]/label/span[2]/input\"}\n  (Session info: headless chrome=115.0.5790.102); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x563e81ead783 <unknown>\n#1 0x563e81bd9917 <unknown>\n#2 0x563e81c1831b <unknown>\n#3 0x563e81c18411 <unknown>\n#4 0x563e81c53674 <unknown>\n#5 0x563e81c38e2d <unknown>\n#6 0x563e81c510d9 <unknown>\n#7 0x563e81c38bd3 <unknown>\n#8 0x563e81c0ca7b <unknown>\n#9 0x563e81c0d81e <unknown>\n#10 0x563e81e6ee98 <unknown>\n#11 0x563e81e72d67 <unknown>\n#12 0x563e81e7d32c <unknown>\n#13 0x563e81e73993 <unknown>\n#14 0x563e81e42827 <unknown>\n#15 0x563e81e978a8 <unknown>\n#16 0x563e81e97a36 <unknown>\n#17 0x563e81ea65f3 <unknown>\n#18 0x7f31d0a67b43 <unknown>","time":{"start":1690985063434,"stop":1690985063434,"duration":0}},{"uid":"b68e710c1593bcaf","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/38//#testresult/b68e710c1593bcaf","status":"broken","statusDetails":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip","time":{"start":1690975072407,"stop":1690975072407,"duration":0}},{"uid":"d2f29f375dd6baa9","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/37//#testresult/d2f29f375dd6baa9","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55b34f07c783 <unknown>\n#1 0x55b34eda8917 <unknown>\n#2 0x55b34edd3978 <unknown>\n#3 0x55b34edd09c9 <unknown>\n#4 0x55b34ee10a13 <unknown>\n#5 0x55b34ee1015f <unknown>\n#6 0x55b34ee07bd3 <unknown>\n#7 0x55b34eddba7b <unknown>\n#8 0x55b34eddc81e <unknown>\n#9 0x55b34f03de98 <unknown>\n#10 0x55b34f041d67 <unknown>\n#11 0x55b34f04c32c <unknown>\n#12 0x55b34f042993 <unknown>\n#13 0x55b34f011827 <unknown>\n#14 0x55b34f0668a8 <unknown>\n#15 0x55b34f066a36 <unknown>\n#16 0x55b34f0755f3 <unknown>\n#17 0x7f282c34ab43 <unknown>","time":{"start":1690974900262,"stop":1690974900262,"duration":0}},{"uid":"d8c03962b25e5b60","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/36//#testresult/d8c03962b25e5b60","status":"broken","statusDetails":"selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"/html/body/app-root/app-auth-screen/app-login-screen/form/div/lta-input[1]/label/span[2]/input\"}\n  (Session info: headless chrome=115.0.5790.102); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n#0 0x5618697cd783 <unknown>\n#1 0x5618694f9917 <unknown>\n#2 0x56186953831b <unknown>\n#3 0x561869538411 <unknown>\n#4 0x561869573674 <unknown>\n#5 0x561869558e2d <unknown>\n#6 0x5618695710d9 <unknown>\n#7 0x561869558bd3 <unknown>\n#8 0x56186952ca7b <unknown>\n#9 0x56186952d81e <unknown>\n#10 0x56186978ee98 <unknown>\n#11 0x561869792d67 <unknown>\n#12 0x56186979d32c <unknown>\n#13 0x561869793993 <unknown>\n#14 0x561869762827 <unknown>\n#15 0x5618697b78a8 <unknown>\n#16 0x5618697b7a36 <unknown>\n#17 0x5618697c65f3 <unknown>\n#18 0x7fdff123bb43 <unknown>","time":{"start":1690973704061,"stop":1690973704061,"duration":0}},{"uid":"cd87386292595e24","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/35//#testresult/cd87386292595e24","status":"broken","statusDetails":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip","time":{"start":1690973260132,"stop":1690973260132,"duration":0}},{"uid":"967fb6f8204ba7a8","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/34//#testresult/967fb6f8204ba7a8","status":"broken","statusDetails":"selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_TIMED_OUT\n  (Session info: headless chrome=115.0.5790.110)\nStacktrace:\n#0 0x5606d2d87783 <unknown>\n#1 0x5606d2ab3917 <unknown>\n#2 0x5606d2aab989 <unknown>\n#3 0x5606d2a9af52 <unknown>\n#4 0x5606d2a9c68b <unknown>\n#5 0x5606d2a9b32d <unknown>\n#6 0x5606d2a9a2ec <unknown>\n#7 0x5606d2a9a25e <unknown>\n#8 0x5606d2a98f1a <unknown>\n#9 0x5606d2a9936a <unknown>\n#10 0x5606d2ab57d0 <unknown>\n#11 0x5606d2b2b885 <unknown>\n#12 0x5606d2b12e02 <unknown>\n#13 0x5606d2b2b0d9 <unknown>\n#14 0x5606d2b12bd3 <unknown>\n#15 0x5606d2ae6a7b <unknown>\n#16 0x5606d2ae781e <unknown>\n#17 0x5606d2d48e98 <unknown>\n#18 0x5606d2d4cd67 <unknown>\n#19 0x5606d2d5732c <unknown>\n#20 0x5606d2d4d993 <unknown>\n#21 0x5606d2d1c827 <unknown>\n#22 0x5606d2d718a8 <unknown>\n#23 0x5606d2d71a36 <unknown>\n#24 0x5606d2d805f3 <unknown>\n#25 0x7fe15aef0b43 <unknown>","time":{"start":1690972630830,"stop":1690972630830,"duration":0}},{"uid":"9e8d9da77059843","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/33//#testresult/9e8d9da77059843","status":"broken","statusDetails":"ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/115.0.5790/chromedriver_linux64.zip","time":{"start":1690971483250,"stop":1690971483250,"duration":0}},{"uid":"7ab8c9fff529ba56","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/32//#testresult/7ab8c9fff529ba56","status":"passed","time":{"start":1682507035320,"stop":1682507042547,"duration":7227}},{"uid":"878bf5e107b0fad1","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/32//#testresult/878bf5e107b0fad1","status":"passed","time":{"start":1682505625513,"stop":1682505632689,"duration":7176}},{"uid":"14f17bc0bd65e0cc","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/31//#testresult/14f17bc0bd65e0cc","status":"passed","time":{"start":1682503856157,"stop":1682503863459,"duration":7302}},{"uid":"e7a5200ff9fb9e9f","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/30//#testresult/e7a5200ff9fb9e9f","status":"passed","time":{"start":1682503294905,"stop":1682503302281,"duration":7376}},{"uid":"53cc6232599be4bd","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/29//#testresult/53cc6232599be4bd","status":"passed","time":{"start":1682502672043,"stop":1682502679233,"duration":7190}},{"uid":"f428f390d2e07c73","reportUrl":"https://MaximKrivenko.github.io/LTA_TESTS/28//#testresult/f428f390d2e07c73","status":"passed","time":{"start":1682500504528,"stop":1682500511732,"duration":7204}}]},"tags":[]},"source":"7dbfeca1528ebbb8.json","parameterValues":[]}